.github/workflows/build-python-client.yml

```diff
@@ -18,24 +18,27 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: 3.10.14
+      - name: install uv
+        uses: yezz123/setup-uv@v4
       - name: Setup Python environment
         run: |
           git status
           make devenv
           source .venv/bin/activate
-      - name: Process tag
+      - name: Git tag to VERSION
         if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
         run: |
           source .venv/bin/activate
           version=$(echo ${{ github.ref }} | grep -oP '(?<=refs/tags/v)\d+\.\d+\.\d+')
           python scripts/is_semantic_version.py "${version}"
-          echo "${version}" > clients/python/client/VERSION
-      - name: Generate client
+          echo "${version}" > clients/python/VERSION
+      - name: Auto-generate python_client and build distributions
         run: |
           source .venv/bin/activate
           cd clients/python
-          make install-dev
-          make dist-ci
+          uv pip install -r requirements/dev.txt
```

Comment:
Question : is uv not used in the male target? That would make it more 1:1 with what you do as dev.
Also there is a GitHub action to install uv

Reply:
@sanderegg what is the "male" target? :-D 
I did not have my PR udpated. I use `uv` everywhere where before was `pip`